lib/kernel/match/match_context.ml

```diff
@@ -68,8 +68,8 @@ let to_json source_path matches =
   let json_matches matches =
     matches
     |> List.map ~f:(fun m ->
-        { m with matched = String.escaped m.matched;
-                 environment = update_environment String.escaped m.environment })
+        { m with matched = m.matched;
+                 environment = update_environment (fun x -> x) m.environment })
```

Comment:
`update_environment` is used elsewhere, and takes a `string -> string` function, so I just replaced it with an identity function. No idea if this is idiomatic.

Reply:
Thanks :-) This reduces to `environment = m.environment` turning the whole `List.map` into identity so can just remove it: see https://github.com/comby-tools/comby/pull/337.

The reason this happened was because I actually only wanted this flavor of excaping for the `-match-only` option, this bit:

https://sourcegraph.com/github.com/comby-tools/comby/-/blob/lib/kernel/match/match_context.ml?L101-102&subtree=true

 and for some reason decided to apply it to the JSON output because I'm a bad programmer.

Reply:
Nice! Thanks for the quick turnaround 😄